src/pages/settings/Profile/ProfilePage.js

```diff
@@ -84,6 +85,10 @@ function ProfilePage(props) {
         },
     ];
 
+    useEffect(() => {
+        App.openProfile(props.currentUserPersonalDetails);
```

Comment:
Why do we want to call openProfile any time `props.currentUserPersonalDetails` changes? 🤔 I assumed we'd only want to call the function the first time we open the profile page - similar to the old `componentDidMount`

Reply:
I had an issue where `myPersonalDetails` was not initialized yet when calling `openProfile` so I thought maybe it would be better to just pass it in 🤔 

But I guess I can return early in `openProfile` if `myPersonalDetails` are not there yet? What do you think?

Reply:
Yes you can add earlier return - that if you do not have myPersonalDetails you should not openProfile

Reply:
Right, but then we have a problem where we call openProfile before our personal details are loaded ...and we never call it again 🤔

Reply:
Sorry for the delay, was ooo friday :D

Oof so I guess if we do the early return (if `props. currentUserPersonalDetails` doesn't exist, return) then we'll have to keep `props.currentUserPersonalDetails` as a dependency, right? I guess I'm ok with that, the current user probably won't be updating their personal details on a different device while they have this page open